### PR TITLE
[docs] add doc for http handler.

### LINKED ISSUE #3143: Document http handler api
**Summary**

Document databend HTTP REST API: Methods, Parameters and samples

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

add doc for HTTP handler.

## Changelog

- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #3143

### CODE REVIEW

**Review by Chasen-Zhang — COMMENTED** [review]
You need to add comments for progress, wall_time_ms, columns and data.

**Comment by youngsofun** [comment]
> You need to add comments for progress, wall_time_ms, columns and data.

@Chasen-Zhang   done

File: website/databend/docs/api/http-handler.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+id: api-http-handler title: HTTP Handler
```
Comment (PsiACE) [inline website/databend/docs/api/http-handler.md:2]:
```suggestion
id: api-http-handler 
title: HTTP Handler
```

**Comment by youngsofun** [comment]
@PsiACE  sorry for not merge your suggestion and rebased.

I formatted the markdown using https://codebeautify.org/markdown-formatter

**Review by Chasen-Zhang — APPROVED** [review]

**Review by PsiACE — APPROVED** [review]
LGTM

**Pull request merged**